### DISCUSSION TITLE
Could you explain the user experience of Go To?

### DISCUSSION THREAD

**Discussion by turkeylurkey** [redhat-developer/lsp4ij#677] [General]
I see a new feature described as "Added support for the IDE's standard Go To Type Declaration using textDocument/typeDefinition." Could you describe what keystroke or sub-menu is normally used to access this IDE feature at the user level? Do you have a link to the IntelliJ docs to show what feature this is related to? 

Previously a change to the "Find Usages" feature caused a problem so I want to test Go To in my environment using the nightly build of lsp4ij.

Thanks!

**Comment by angelozerr** [comment]
@SCWells72 did an amazing work about that. Now the 2 standards actions should be available (ifyour language server supports implementations or typeDefinitions):

![image](https://github.com/user-attachments/assets/bf3bed6c-6511-46ef-a5bc-c267f00ec9da)

I think it would be nice to add in https://github.com/redhat-developer/lsp4ij/blob/main/docs/UserGuide.md a Shortcut section to clarify that.

> Previously a change to the "Find Usages" feature caused a problem so I want to test Go To in my environment using the nightly build of lsp4ij.

Which problem do you have? And which version of LSP4IJ causes this problem 0.81 ? nighthy?

Please note that I have the intention to release (perhaps this week?) a new release of 0.9.0 to fix thebug https://github.com/redhat-developer/lsp4ij/issues/660 introduced in 0.8.1

This release will hosts all great stuff that @SCWells72 did (perhaps my work on JSON Schema for user defined ls) and the fix.

**Reply by turkeylurkey** [reply]
Thank you for the help. I found those actions on the Go To menu which I previously have not needed to use. It looks like everything is working. 👍
